scm/github/changeset.go

```diff
@@ -8,12 +8,13 @@ import (
 
 	"github.com/sirupsen/logrus"
 
+	api "github.com/go-vela/server/api/types"
 	"github.com/go-vela/types/library"
 	"github.com/google/go-github/v61/github"
 )
 
 // Changeset captures the list of files changed for a commit.
-func (c *client) Changeset(ctx context.Context, u *library.User, r *library.Repo, sha string) ([]string, error) {
+func (c *client) Changeset(ctx context.Context, u *library.User, r *api.Repo, sha string) ([]string, error) {
```

Comment:
question: could we get rid off `u *library.User` since that is likely coming from `r.GetOwner()` now? i know there's some other place in /scm/*, so i won't repeat the question in those spots.